src/services/document-service.spec.js

```diff
@@ -23,6 +27,42 @@ describe('document-service', () => {
     await pruneTestEnvironment(container);
   });
 
+  describe('createDocumentRevision', () => {
+    let key;
+    beforeEach(async () => {
+      const doc = {
+        title: 'Title',
+        slug: 'my-doc',
+        namespace: 'articles',
+        language: 'en',
+        sections: [
+          {
+            key: uniqueId.create(),
+            type: 'image',
+            content: {
+              sourceType: SOURCE_TYPE.internal,
+              sourceUrl: 'media/image-1.png',
+              effect: {
+                sourceType: SOURCE_TYPE.internal,
+                sourceUrl: 'media/image-2.png'
+              }
+            }
+          }
```

Comment:
Maybe overkill, but how about just adding a second section here, only to have the test include/prove the fact that all sections are iterated?

Reply:
OK, good idea

Reply:
Done!